### PR TITLE
GH alternative route

### PR DESCRIPTION
**Pull request by crixal**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3157)

### CODE REVIEW

**Review by JustinPihony — APPROVED** [review]

**Comment by JustinPihony** [comment]
@crixal Can this one be merged?

**Comment by alex-vv** [comment]
test!

**Comment by alex-vv** [comment]
test!

**Comment by alex-vv** [comment]
test!

**Comment by alex-vv** [comment]
test!

**Comment by alex-vv** [comment]
test!

**Comment by alex-vv** [comment]
test!

File: src/main/scala/beam/router/RoutingWorker.scala [hunk 1]
```diff
@@ -478,7 +484,8 @@ object RoutingWorker {
           0,
           unbecomeDriverOnCompletion = true
         )
-      )
+      ),
+      Some("R5")
```
Comment (nikolayilyin) [inline src/main/scala/beam/router/RoutingWorker.scala]: if it is not used in filtering or any kind of logic, I would use "Bushwalking" here as route type

**Review by nikolayilyin — APPROVED** [review]
LGTM

**Comment by nikolayilyin** [comment]
test!

**Pull request merged**